### PR TITLE
docs(chart): add a11y yml file

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: components/chart/accessibility/wai-aria-support.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: Wai-Aria Support
+page_title: Telerik UI for Blazor Chart Documentation | Chart  Accessibility
```
Comment (ikoevska) [inline components/chart/accessibility/wai-aria-support.md:3]: Probably an issue with the template used for generating this page. There are two spaces between the component name and the word "accessibility".

```suggestion
page_title: Telerik UI for Blazor Chart Documentation | Chart Accessibility
```

File: components/chart/accessibility/wai-aria-support.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+# Blazor Chart Accessibility
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+
+
+Out of the box, the Telerik UI for Blazor Chart provides extensive accessibility support and enables users with disabilities to acquire complete control over its features.
+
+
+The Chart is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.2  AA](https://www.w3.org/TR/WCAG22/) standards](https://www.w3.org/TR/WCAG22/) and [Section 508](http://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
```
Comment (ikoevska) [inline components/chart/accessibility/wai-aria-support.md:19]: Again, a weird link issue with the template used for generating the page. Note that it affects multiple live pages.
```suggestion
The Chart is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.2 AA](https://www.w3.org/TR/WCAG22/) standards and [Section 508](http://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
```

**Review by ikoevska — COMMENTED** [review]
Note that a fix to the template for generating these pages is needed.

**Comment by dimodi** [comment]
> Note that a fix to the template for generating these pages is needed. I have also noted it in #1839

Yes, here is the PR: https://github.com/telerik/web-components-ux/pull/968

**Review by ikoevska — COMMENTED** [review]
Unfortunately the link issue still stands. Please, ping me when the [PR to the template](https://github.com/telerik/web-components-ux/pull/968) gets fixed and the article is re-generated. 

There are two ways to go about this:

Option A) We merge as is. After all, all other accessibility articles have the same issue. We then re-generate after the PR to the template gets merged.
Option B) We delay the merge until the PR to the template gets fixed. However, this will probably result in every wai-aria page getting re-generated as well. (It might be better to see this in a separate PR for history purposes though).

It depends on how urgent this a11y article is. If urgent, I am inclined to get this merged as is, if we get a tracking issue for re-generating content once the template is fixed.

**Comment by dimodi** [comment]
This PR contains a fix, which will affect another PR for the TreeList, so we will merge it and take care of the re-generation.

**Review by xristianstefanov — APPROVED** [review]

**Pull request merged**